resources/js/frontend/wc-payplug-oney_x3_with_fees-blocks.js

```diff
@@ -0,0 +1,107 @@
+import {__} from '@wordpress/i18n';
+import {registerPaymentMethod} from '@woocommerce/blocks-registry';
+import {decodeEntities} from '@wordpress/html-entities';
+import {getSetting} from '@woocommerce/settings';
+
+
+
+const settings = getSetting('oney_x3_with_fees_data', {});
+const defaultLabel = __('Gateway method title', 'payplug');
+const label = decodeEntities(settings?.title) || defaultLabel;
+
+let Content;
+
+const translations = settings?.translations;
+
+const allowed_country_codes = settings?.requirements.allowed_country_codes;
+
+const oney_response = settings?.oney_response;
+
+if (typeof oney_response.x3_with_fees !== 'undefined') {
+	var down_payment_amount = parseFloat(settings?.oney_response['x3_with_fees']['down_payment_amount']) ;
+	var total_price_oney = down_payment_amount;
+	settings?.oney_response['x3_with_fees']['installments'].forEach((amount) => {
+
+		total_price_oney += parseFloat(amount['amount']);
+
+	});
+
+	Content = (e) => {
```

Comment:
Couldn't we have a generic OneySimulation Component with the values as parameter which would be used on the render?